src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final StorageDao storageDao = new StorageDaoImpl();
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void register_successful_Ok() {
+        User actual = registrationService.register(new User("Alice123", "123456", 19));
```

Comment:
move user to the class field and setup in the @BeforeEach annotation method

Reply:
for specific cases like short login, replace the login in the test method with the required one